index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Carrusel(mecostomásdeloqueesperaba)</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="script.js" defer></script>
+</head>
+<body>
+
+    <div class="carrusel">
+        
+        <div class="control atras">
+            <img id="atras" src="assets/AtrasIzq.png" alt="Retroceder" loading="lazy">
+        </div>
```

Comment:
Agregarle un evento de click a un div no es lo adecuado en este caso. Para los elementos de navegación la etiqueta más adecuada es un <button>

```suggestion
        <button class="control atras">
            <img id="atras" src="assets/AtrasIzq.png" alt="Retroceder" loading="lazy">
        </button>
```